ARImageTrackingExamples/ARImageTrackingExamples/Helpers/UIViewController+helpers.swift

```diff
@@ -9,8 +9,15 @@ extension UIViewController {
 
 extension Float {
     static let smallOffset: Float = 0.01
+    static let rightAngle: Float = .pi / 2
 }
 
 extension CGFloat {
     static let slightyTransparent: CGFloat = 0.95
+    static let half: CGFloat = 0.5
+    static let third: CGFloat = 0.75
```

Comment:
third ? 3/4 no ?

Reply:
Yes, we should rename it

Reply:
I'd be grateful for ideas though. Should we call it `threeQuaters` then? or since it's used to set xOffset based on image.width we could name it like : `xOffsateRate` ?

Reply:
I don't think naming threeQuarters (or half) explains much more than 0.75 or 0.5 tbh... also xOffsetRate sounds vague (offset of what?). I'd probably just inline the value 0.75